doc/content/msr/msfr/griffin_pgh_model.md

```diff
@@ -334,33 +333,80 @@ symmetry and adiabatic boundaries.
 
 Legacy syntax for the energy equation is included [here](griffin_pgh_model_legacy.md).
 
+## Conservation of delayed neutron precursors
+
+The steady-state conservation of delayed neutron precursors (DNP) can be expressed as,
+\begin{equation}
+  \nabla \cdot (\vec{u}c_i) -  \nabla \cdot \epsilon_c \nabla c_i - \lambda_i c_i = \beta_i f,\quad i=1,\cdots,I,
+  \label{eq:dnp}
+\end{equation}
+where $c_i$, $\lambda_i$ and $\beta_i$ are the concentration, the decay constant and
```

Comment:
I am afraid there is a mistake in the input: the fission source needs to be scaled with k-eff for eigenvalue calculations because the fission source provided by Griffin does not contain this scaling factor. Griffin input will need to have
```
  [keff]
    type = MultiAppPostprocessorTransfer
    to_multi_app = ext_dnp
    from_postprocessor = eigenvalue
    to_postprocessor = keff
  []
```
to transfer the eigenvalue or keff to NS and NS needs to use it. Another option is to make Griffin evaluate the fission source with the k-eff scaling, which will require a new control parameter for this and some code enhancement.

Reply:
We ve done this manually so far actually

Reply:
How did you do that? You have to know the converged k-eff, which is the value these inputs are solving for.

Reply:
Oh yeah we did not do that here. 
We scaled the power distribution in the transient inputs.

Reply:
This is issue. I tried not having the scaling locally with a Griffin test. The code runs but converges to a different eigenvalue. We can add the transfer to transfer the keff to NS, but the action needs to be able to use it in the kernel.

Reply:
Fixing this will take a lot of work to regold. I think we may combine it with another change

Reply:
I am thinking to have fission source evaluated by Griffin always scaled with keff. If we have that, the input will need no change. Regolding will be needed but that can be done quickly.

Reply:
(This is Mauricio) Sounds good. Please keep in mind that the MSFR models have been tested and checked against OpenFOAM ones and the accuracy physics in the model is carefully analyzed. If regolding needs to be done, please ensure that the solution fields (velocity, temperature, flux, etc.) do not change

Reply:
I am not expecting big changes on fluid side because this is affecting DNP directly only. It will take me some time on finishing this in Griffin because it affects how Griffin treats this in the past and can result careful regolding.